WalletWasabi/Helpers/Startup/LinuxStartupChecker.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Helpers
+{
+	public static class LinuxStartupChecker
+	{
+		private static readonly string DesktopFileContent = string.Join(
```

Comment:
Maybe one can use the terminology from xUnit:

```suggestion
		private static readonly string ExpectedFileContent = string.Join(
```